### DISCUSSION TITLE
Licensing and Copyright for diagrams

### DISCUSSION THREAD

**Discussion by talal** [xy-241/CS-Notes#51] [General]
Hi,

Thank you for publishing your notes 😊

I wanted to ask you if I could use your diagrams in my notes, with proper attribution of course?

Perhaps it would be a good idea for you to add a license to your notes and diagrams, something like the [Creative Commons](https://creativecommons.org/licenses/by/4.0/) license so that people know the legal status on reusing your material.

**Comment by xy-241** [comment]
Yup sure! I have put my work under the CC BY 4.0 license. Thanks for the heads up!

<img width="564" alt="Screenshot 2024-08-22 at 9 57 45 PM" src="https://github.com/user-attachments/assets/33158ef5-7c9d-43fa-baea-f1baafd0fe80">

**Reply by talal** [reply]
Thanks a lot, much appreciated :)
